packages/editor/src/components/table-of-contents/panel.js

```diff
@@ -57,7 +57,7 @@ function TableOfContentsPanel( { headingCount, paragraphCount, numberOfBlocks }
 }
 
 export default withSelect( ( select ) => {
-	const { getGlobalBlockCount } = select( 'core/editor' );
+	const { getGlobalBlockCount } = select( 'core/block-editor' );
```

Comment:
`getGlobalBlockCount` sounds like something that should consider ALL loaded editor instances in a view (as opposed to `getBlockCount` which just would consider count of blocks for just that editor instance.

Reply:
I think the difference at the moment between the two is whether to count the inner blocks or not.

Reply:
Hmm unfortunate name then. It does present potential naming collision/confusion if in the future there's some way of counting blocks for all editor instances in a view.

Reply:
What kind of use-case do you imagine for having a count of all blocks across editors?

Reply:
None offhand besides more of an informational stats like view to the end-user.  I don't think this is critical to change, but the fact that it's not immediately apparent what this function represents is a bit unfortunate (the whole "naming is hard" kind of thing ;) ).  "Global" is a fairly broad term.